common/services/user-locations.test.js

```diff
@@ -0,0 +1,94 @@
+const proxyquire = require('proxyquire')
+
+const referenceDataStub = {
+  getLocations: async args => {
+    return [{ id: 'test' }, { id: 'test2' }]
+  },
+}
+
+const userLocationsService = proxyquire('./user-locations', {
+  './reference-data': referenceDataStub,
+})
+
+const { AUTH_PROVIDERS, NOMIS_ELITE2_API } = require('../../config')
```

Comment:
I think it's better to stub these using `proxyquire` rather than load them into the test file.

This is the problem we have in the other services at the moment and is causing #82. I think it's what is causing the tests to fail on CI too.

Reply:
👍